RunLog/RunLog/Sources/Presentation/Log/View/Calendar/CalendarViewCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CalendarViewCell.swift
+//  RunLog
+//
+//  Created by 신승재 on 3/16/25.
+//
+
+import UIKit
+import SnapKit
+import Then
+
+class CalendarViewCell: UICollectionViewCell {
+    
+    static let identifier = "DayCell"
+    
+    var dayLabel = UILabel().then {
+        $0.attributedText = .RLAttributedString(
+            text: "1",
+            font: .Label2,
+            color: .Gray000
+        )
+        $0.textAlignment = .center
```

Comment:
```suggestion
        $0.attributedText = .RLAttributedString(
            text: "1",
            font: .Label2,
            align: .center
        )
```
RLAttributedString 속성에 align있습니다! 그리고 color는 도로시의 PR에서 기본값이 Gray000으로 바뀌어서 지정안해주셔도 됩니다

Reply:
제가 다시 black으로 바꿧습니다 충돌나서,,이거는 17일 오전에 한번 손보죠!
배경이 검은색이라 기본 폰트가 .gray000이 더 많이 쓰이는것 같아서 일단 바꿔뒀었는데...confilct나서 수정했습니다

Reply:
요런 경우 저는 색을 명시하는게 추후에 수정할때 용이하다고 생각해서 작성해놓았습니다 !